frontend/homework/1/frontend-homework-1/index.html

```diff
@@ -0,0 +1,165 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <link rel="stylesheet" href="styles.css">
+  <!-- <link rel="stylesheet" href="reset.css"> -->
+    <title>kickdrum.com</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta charset="UTF-8">
+  <meta name="description" content="Kickdrum website">
+  
+
+</head>
+
+<header>
+
+<nav >
+  <div class="nav-top">
+    <ul id="top-nav">
+      <li class="nav-bar"><a href="#" tabindex="2">Service </a></li>
+      <li class="nav-bar"><a href="#"tabindex="1">Results</a></li>
+      <li class="nav-bar"><a href="#">Team</a></li>
+      <li id="title-nav"><h1>KickDrum</h1></li>
+      <li class="nav-bar" style="float:right"><a href="#">Resource</a></li>
+      <li class="nav-bar" style="float:right"><a href="#">Contact</a></li>
+      <nav>
+        <input  class="nav-bar" style="float:right" type="text"   placeholder="Search.." title="Type of Course">
+      </nav>
+    </ul>
+  </div>
+  </nav>
+    <div class="hire">
+    <h1>Now we are hiring <b>exceptional enterprise-grade engineer</b></h1>
+    </div>
+</ul>
+
+</header>
+
+<main>
+    <!-- <div class="image">  
+                  <img src="https://miro.medium.com/v2/resize:fit:1200/1*XWDUzu1Y708Q4IkG4o4UMw.png" alt="Kickdrum Logo" width="154" height="142">
+    </div>
+             -->
+
+            <!-- <iframe src="https://www.kickdrum.com/" title="KickDrum Website"  target="_parent"></iframe> -->
+
+            <!-- <div>
+              <div class="image">  
+                <img src="https://miro.medium.com/v2/resize:fit:1200/1*XWDUzu1Y708Q4IkG4o4UMw.png" alt="Kickdrum Logo" width="154" height="142">
+  </div> -->
```

Comment:
Remove unnecessary code if not required
If you did it intentionally just to follow requirements, then write meaningful comments instead of code comment